src/Microsoft.DotNet.Interactive.Documents.Tests/NotebookParserServerTests_ObjectInterface.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.Documents.ParserServer;
+using Xunit;
+
+namespace Microsoft.DotNet.Interactive.Documents.Tests
+{
+    public class NotebookParserServerTests_ObjectInterface
+    {
+        [Theory]
+        [InlineData(DocumentSerializationType.Dib, "#!csharp\nvar x = 1;")]
+        [InlineData(DocumentSerializationType.Ipynb, @"{""cells"":[{""cell_type"":""code"",""source"":[""var x = 1;""]}]}")]
```

Comment:
Think these names are inconsistent with what we're calling "CodeSubmission" and "Jupyter" elsewhere.

Reply:
You mean for the enum `DocumentSerializationType`?

Would you prefer this?

``` csharp
public enum DocumentSerializationType
{
    CodeSubmission,
    Jupyter,
}
```

Reply:
That looks good.